apps/etterlatte-beregning/src/main/kotlin/beregning/BeregnBarnepensjonService.kt

```diff
@@ -64,7 +64,15 @@ class BeregnBarnepensjonService(
             beregningsGrunnlagService.hentBarnepensjonBeregningsGrunnlag(behandling.id, bruker)
         )
 
-        val trygdetid = trygdetidKlient.hentTrygdetid(behandling.id, bruker)
+        val trygdetid = try {
+            trygdetidKlient.hentTrygdetid(behandling.id, bruker)
+        } catch (e: Exception) {
+            logger.warn(
+                "Kunne ikke hente ut trygdetid for behandlingen med id=${behandling.id}. " +
+                    "Dette er ikke kritisk siden vi ikke har krav om trygdetid enda."
```

Comment:
Pirk, og kan sikkert vente til neste iterasjon, men vil ha med e også i warn-en her. Eller ha ein meir spesifikk catch for akkurat den forventa-isj exceptionen.